build.gradle

```diff
@@ -8,13 +8,17 @@ plugins {
 group = 'org.alvearie.hri'
 version = '1.2.0'
 
-//TODO update for GitHub actions
-ext.branch = System.getenv('TRAVIS_BRANCH') != null
-        ? System.getenv('TRAVIS_BRANCH')
+ext.branch = System.getenv('ACTIONS_BRANCH') != null
+        ? System.getenv('ACTIONS_BRANCH')
         : grgit.open(currentDir: project.rootDir).branch.current.name
-if ("${branch}" != 'master') {
-    version = "${version}-${branch}-SNAPSHOT"
+if (System.getenv('ACTIONS_TAG') == null || System.getenv('ACTIONS_TAG') == "") {
+    version = "${branch}-SNAPSHOT"
+} else if (System.getenv('ACTIONS_TAG') == "v${version}") {
+    version = "${version}"
+} else {
+    throw new InvalidUserDataException(String.format("The tag '%s' does not match with the current release version '%s'",System.getenv('TRAVIS_TAG'),"${version}"));
```

Comment:
Q. Is this change correct. The previous code on the support-1.x branch was:

```
ext.branch = System.getenv('TRAVIS_BRANCH') != null
        ? System.getenv('TRAVIS_BRANCH')
if ("${branch}" != 'master') {
    version = "${version}-${branch}-SNAPSHOT"
```

...which did not seem correct to me.

Reply:
I think what you changed it to is correct.